capreolus/searcher/special.py

```diff
@@ -0,0 +1,196 @@
+import os
+import gdown
+from pathlib import Path
+from collections import defaultdict
+
+from capreolus import ConfigOption, Dependency
+from capreolus.utils.loginit import get_logger
+
+from . import Searcher
+from .anserini import BM25
+
+logger = get_logger(__name__)
+
+SUPPORTED_TRIPLE_FILE = ["small", "large.v1", "large.v2"]
+
+
+def get_file_line_number(fn):
+    return int(os.popen(f"wc -l {fn}").readline().split()[0])
```

Comment:
Is this better than something in Python like `sum(1 for line in open(fn, "rt"))` ?

Reply:
was using this as it seems to be slightly faster, say on the msmarco psg `collection.tsv`:
```
Use wc -l:       0.7997s
Number of Lines: 8841823
Use python way:  3.8262s
Number of Lines: 8841823
```
But just removed this function as seems it's no long been used anywhere :P